fixed_expressions.csv

```diff
@@ -115,11 +115,9 @@ char,jyutping
 不費吹灰之力,bat1 fai3 ceoi1 fui1 zi1 lik6
 不食人間煙火,bat1 sik6 jan4 gaan1 jin1 fo2
 且聽下回分解,ce2 ting1 haa6 wui4 fan1 gaai2
-世上無難事，只怕有心人,sai3 soeng5 mou4 naan4 si6 zi2 paa3 jau5 sam1 jan4
 亞茂整餅，冇嗰樣整嗰樣,aa3 mau6 zing2 beng2 mou5 go2 joeng2 zing2 go2 joeng2
 人一世，物一世,jan4 jat1 sai3 mat6 jat1 sai3
 人不可貌相，海水不可斗量,jan4 bat1 ho2 maau6 soeng3 hoi2 seoi2 bat1 ho2 dau2 loeng6
-人不爲己，天誅地滅,jan4 bat1 wai6 gei2 tin1 zyu1 dei6 mit6
```

Comment:
Why is this deleted btw

Reply:
Should be delete the one in words.csv and keep the one in fixed_expressions

Reply:
Do you mind making the de-duper just output a warning for now, and then killing the last de-dupe commit? Then we can manually fix them incrementally. Otherwise, this would undo quite a bit of cleanup I just had these two weeks :/

Reply:
OK